src/DynamoCore/DynamoCore.csproj

```diff
@@ -431,6 +427,12 @@ limitations under the License.
     <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />
   </Target>
   <Target Name="AfterBuild" Condition=" '$(OS)' != 'Unix' ">
+    <ItemGroup>
+      <ASMLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
```

Comment:
As discussed, this is still needed for the `protogeometry.config`, customization DLL, etc., correct?

Reply:
That's right, ProtoGeometry.config and the 2 XML files.